### PR TITLE
Add Windows Terminal

### PR DESCRIPTION
**Pull request by nimajneBG**

### CODE REVIEW

File: terminal/windows-terminal/dark-hard.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "repose dark hard",
```
Comment (l4ssc) [inline terminal/windows-terminal/dark-hard.json:2]: Please format the name as Repose (Variant), so for example Repose (Dark Hard) in this case.

**Comment by l4ssc** [comment]
lgtm outside of the naming of the theme (which should be a very simple change), thanks for the contribution! 👍

**Comment by nimajneBG** [comment]
Changed the names according to your suggestion/wish

**Comment by l4ssc** [comment]
🎉

**Pull request merged**